x-pack/plugins/uptime/server/lib/alerts/duration_anomaly.ts

```diff
@@ -63,62 +61,74 @@ const getAnomalies = async (
   );
 };
 
-export const durationAnomalyAlertFactory: UptimeAlertTypeFactory<ActionGroupIds> = (
-  _server,
-  _libs,
-  plugins
-) =>
-  uptimeAlertWrapper<ActionGroupIds>({
-    id: 'xpack.uptime.alerts.durationAnomaly',
-    name: durationAnomalyTranslations.alertFactoryName,
-    validate: {
-      params: schema.object({
-        monitorId: schema.string(),
-        severity: schema.number(),
-      }),
+export const durationAnomalyAlertFactory: UptimeAlertTypeFactory = (_server, _libs, plugins) => ({
+  id: 'xpack.uptime.alerts.durationAnomaly',
+  name: durationAnomalyTranslations.alertFactoryName,
+  validate: {
+    params: schema.object({
+      monitorId: schema.string(),
+      severity: schema.number(),
+    }),
+  },
+  defaultActionGroupId: DURATION_ANOMALY.id,
+  actionGroups: [
+    {
+      id: DURATION_ANOMALY.id,
+      name: DURATION_ANOMALY.name,
     },
-    defaultActionGroupId: DURATION_ANOMALY.id,
-    actionGroups: [
-      {
-        id: DURATION_ANOMALY.id,
-        name: DURATION_ANOMALY.name,
-      },
-    ],
-    actionVariables: {
-      context: [],
-      state: [...durationAnomalyTranslations.actionVariables, ...commonStateTranslations],
-    },
-    minimumLicenseRequired: 'basic',
-    async executor({ options, uptimeEsClient, savedObjectsClient, dynamicSettings }) {
-      const {
-        services: { alertInstanceFactory },
-        state,
-        params,
-      } = options;
+  ],
+  actionVariables: {
+    context: [],
+    state: [...durationAnomalyTranslations.actionVariables, ...commonStateTranslations],
+  },
+  minimumLicenseRequired: 'platinum',
```

Comment:
I did add that - AFAIK we should restrict this to platinum because it relies on ML, a platinum feature. I didn't write this alert though, so I'll admit to that being a big assumption on my part. cc @shahzad31